isis/src/control/objs/ControlNet/ControlNet.cpp

```diff
@@ -453,12 +476,11 @@ namespace Isis {
         ControlMeasure *cm = point->GetMeasure(i);
         if (!cm->IsIgnored()) {
           QString sn = cm->GetCubeSerialNumber();
-          ControlCubeGraphNode *neighborNode = (*cubeGraphNodes)[sn];
 
-          if (neighborNode != node) {
-            node->addConnection(neighborNode, point);
-            neighborNode->addConnection(node, point);
-          }
+          ImageConnection connection = boost::add_edge(m_vertexMap[serial],
```

Comment:
It's probably good to add a comment everywhere that we add edges about how it doesn't always add an edge, that we are exploiting the fact that using a set forces unique edges and in the case that the edge exists, this just returns it.

Reply:
That's a good point, thank you! I will add that comment.

Reply:
:+1: Okay, added this everywhere an add is done.